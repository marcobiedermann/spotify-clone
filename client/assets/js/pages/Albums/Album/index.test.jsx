import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import { AlbumPage } from '.';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <MemoryRouter>
        <AlbumPage />
      </MemoryRouter>,
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
