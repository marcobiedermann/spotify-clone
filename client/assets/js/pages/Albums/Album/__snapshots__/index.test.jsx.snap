// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="album-page"
>
  <div
    className="error"
  >
    
  </div>
  <div>
    <figure>
      <figcaption>
        <h3>
          <a
            href="/albums/"
            onClick={[Function]}
          >
            
          </a>
        </h3>
      </figcaption>
    </figure>
    <table>
      <tbody />
    </table>
  </div>
</div>
`;
